apps/air-discount-scheme/backend/src/app/modules/discount/discount.service.ts

```diff
@@ -1,14 +1,17 @@
 import { uuid } from 'uuidv4'
 import { Inject, Injectable, CACHE_MANAGER } from '@nestjs/common'
 
-import { Discount } from './discount.model'
-import { Flight } from '../flight'
+import { Discount, ExplicitCode } from './discount.model'
+import { Flight } from '../flight/flight.model'
```

Comment:
Weird Bug Note: In places affected by the SequelizeModule directly, the models themselves must not come from the barrel files lest they run the risk of being undefined.